install/cupy_builder/_compiler.py

```diff
@@ -64,7 +64,29 @@ def _nvcc_gencode_options(cuda_version: int) -> List[str]:
         #   https://docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#options-for-steering-gpu-code-generation
 
         aarch64 = (platform.machine() == 'aarch64')
-        if cuda_version >= 11040:
+        if cuda_version >= 11080:
+            arch_list = [('compute_35', 'sm_35'),
+                         ('compute_37', 'sm_37'),
+                         ('compute_50', 'sm_50'),
```

Comment:
Is it intentional to exclude `('compute_52', 'sm_52')`?

Reply:
Thanks, that's my mistake..